components/feature/syncedtabs/src/main/java/mozilla/components/feature/syncedtabs/interactor/DefaultInteractor.kt

```diff
@@ -33,11 +27,6 @@ internal class DefaultInteractor(
     }
 
     override fun onRefresh() {
-        CoroutineScope(coroutineContext).launch {
-            accountManager.withConstellation {
-                refreshDevicesAsync()
-            }
-            accountManager.syncNowAsync(SyncReason.User, true)
-        }
+        controller.syncAccount()
```

Comment:
So if i'm reading this correctly, this is just a refactor? You were already requesting a syncNow before this PR.

Reply:
Before we were calling `syncTabs` which pulls the tabs from the last sync. Now, we're calling `syncAccount` which is what we were doing when a user activated sync from the UI. When the account syncs and the account manager is idle, we do the `syncTabs` call.